### PR TITLE
Update MapView doc pattern

### PR DESCRIPTION
**Pull request by shubham7109**
PR to address the pattern used for the compose MapView doc. 
 
Related issue: `#3164`

### CODE REVIEW

**Review by hud10837 — COMMENTED** [review]
This is a good change, but I think we can probably also remove the references to `composable [MapView]` from all of the parameters here. These are parameters on the composable MapView, so there's no need to link to it here. I'm not sure what we can replace it with though, perhaps just:

```
@param onScale invoked when a user performs a pinch gesture on the MapView
```

**Comment by shubham7109** [comment]
@hud10837 Sure, I think it would be fine to remove the in-line references (`[]`) but I would still think there is value in using the prefix (view-based/composable) when `MapView` is referenced. I believe it will help clarify which type of MapView the doc is referring to. 

Adding on your suggestion would you say it would be acceptable to use:
```
* @param onScale invoked when a user performs a pinch gesture on the composable MapView
```

**Comment by hud10837** [comment]
I think this would be acceptable 👍

**Comment by shubham7109** [comment]
Updated PR, ready for your review @hud10837

**Review by hud10837 — APPROVED** [review]
Thanks for those changes 👍

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt [hunk 1]
```diff
@@ -56,17 +56,17 @@ import com.arcgismaps.mapping.view.geometryeditor.GeometryEditor
...
- * @param locationDisplay the [LocationDisplay] used by the composable [com.arcgismaps.toolkit.geocompose.MapView]
- * @param geometryEditor the [GeometryEditor] used by the composable [com.arcgismaps.toolkit.geocompose.MapView] to create and edit geometries by user interaction.
- * @param mapViewInteractionOptions the [MapViewInteractionOptions] used by this composable [com.arcgismaps.toolkit.geocompose.MapView]
- * @param viewLabelProperties the [ViewLabelProperties] used by the composable [com.arcgismaps.toolkit.geocompose.MapView]
- * @param selectionProperties the [SelectionProperties] used by the composable [com.arcgismaps.toolkit.geocompose.MapView]
- * @param grid represents the display of a coordinate system [Grid] on the composable [com.arcgismaps.toolkit.geocompose.MapView]
+ * @param arcGISMap the [ArcGISMap] to be rendered by this composable MapView
+ * @param graphicsOverlays the [GraphicsOverlayCollection] used by this composable MapView
+ * @param locationDisplay the [LocationDisplay] used by the composable MapView
+ * @param geometryEditor the [GeometryEditor] used by the composable MapView to create and edit geometries by user interaction.
```
Comment (gunt0001) [inline toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:65]: nitpick - remove full stop:
```suggestion
 * @param geometryEditor the [GeometryEditor] used by the composable MapView to create and edit geometries by user interaction
```

**Review by gunt0001 — APPROVED** [review]
@shubham7109 - looks good, one minor comment

**Pull request merged**